persistent-template/Database/Persist/TH.hs

```diff
@@ -195,22 +196,23 @@ embedEntityDefsMap rawEnts = (embedEntityMap, noCycleEnts)
       let entName = entityHaskell entDef
       in  entDef { entityFields = map (breakCycleField entName) $ entityFields entDef }
 
-    breakCycleField entName f@(FieldDef { fieldReference = EmbedRef em }) =
-      f { fieldReference = EmbedRef $ breakCycleEmbed [entName] em }
-    breakCycleField _ f = f
+    breakCycleField entName f = case f of
+      FieldDef { fieldReference = EmbedRef em } ->
+        f { fieldReference = EmbedRef $ breakCycleEmbed [entName] em }
+      _ ->
+        f
 
     breakCycleEmbed ancestors em =
-        em { embeddedFields = map (breakCycleEmField $ emName : ancestors)
-                                  (embeddedFields em)
-           }
+      em { embeddedFields = breakCycleEmField (emName : ancestors) <$> embeddedFields em
+         }
       where
         emName = embeddedHaskell em
 
     breakCycleEmField ancestors emf = case embeddedHaskell <$> membed of
-        Nothing -> emf
-        Just embName -> if embName `elem` ancestors
-          then emf { emFieldEmbed = Nothing, emFieldCycle = Just embName }
-          else emf { emFieldEmbed = breakCycleEmbed ancestors <$> membed }
+      Nothing -> emf
+      Just embName -> if embName `elem` ancestors
+        then emf { emFieldEmbed = Nothing, emFieldCycle = Just embName }
+        else emf { emFieldEmbed = breakCycleEmbed ancestors <$> membed }
```

Comment:
The convention in this package is to use 4 space indent, not 2 space. Please revert all 2 space indentation changes and  use 4 space indent on the project.